src/styles/index.styl

```diff
@@ -1,4 +1,5 @@
 @require 'cozy-ui'
+@require 'utilities/*'
```

Comment:
Since Harvest use only utilities classes from cozy-ui and cozy-ui have some issues about having all of them, we have to include them explicitly in the application